src/views/bookings/PendingBookings.js

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable no-nested-ternary */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import Carousel from 'react-bootstrap/Carousel';
+import { Link } from 'react-router-dom';
+import { Card } from 'react-bootstrap';
+import { getPendingBookings, approveBooking, rejectBooking } from '../../actions/bookingActions';
+import Breadcrumbs from '../../components/global/Breadcrumbs';
+import PageLoading from '../../components/global/PageLoading';
+import isAuthenticated from '../../helpers/isAuthenticated';
+import AlertComponent from '../../components/global/AlertComponent';
+import Confirm from '../../components/global/Confirm';
+
+class PendingBookings extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: true,
+      userId: null,
+      operation: null,
+      isProcessing: false,
+    };
+  }
+
+  componentDidMount = async () => {
+    const { getPendingBookings } = this.props;
+    const userInfo = isAuthenticated();
```

Comment:
Add a new line

Reply:
Added line, not sure why it's not reflecting